iffe's check for a function being in libraries assumes -O0, but we have -O1 as default which optimises away the check
diff -ur graphviz-2.40.1.orig/iffe graphviz-2.40.1/iffe
--- graphviz-2.40.1.orig/iffe	2016-10-08 17:56:00.000000000 +0000
+++ graphviz-2.40.1/iffe	2019-04-01 10:29:37.925505155 +0000
@@ -3122,7 +3122,7 @@
 extern int $v();
 _END_EXTERNS_
 #endif
-static _IFFE_fun i=(_IFFE_fun)$v;int main(){return(i==0);}
+_IFFE_fun i=(_IFFE_fun)$v;int main(){return(i==0);}
 "
 				d=-D_IFFE_extern
 				if	$cc -c $tmp.c <&$nullin >&$nullout
